src/components/container/ConnectFitbit/ConnectFitbit.stories.tsx

```diff
@@ -1,48 +1,103 @@
 ﻿import React from "react"
-import { ComponentStory, ComponentMeta } from "@storybook/react"
 import ConnectFitbit, { ConnectFitbitProps } from "./ConnectFitbit"
 import Card from "../../presentational/Card"
 import Layout from "../../presentational/Layout"
+import { Meta, StoryObj } from "@storybook/react/*";
 
-export default {
+
+const meta: Meta<typeof ConnectFitbit> = {
 	title: "Container/ConnectFitbit",
 	component: ConnectFitbit,
 	parameters: {
 		layout: 'fullscreen',
+		docs: {
+			//TODO: Description: <Description />
```

Comment:
Since this is optional and probably applies to all of our stories, can we remove the TODO comment for now?  Or go ahead and implement the description?

Reply:
I'm removing them for now. I will clean up the rollup in a new PR once this one is merged